Fix bug where lproj directories aren't added to project

efirestone opened the pull request:
https://github.com/CocoaPods/CocoaPods/issues/4762

Root of the issue was that the resources were being added to the xcodeproj in all-lowercase, and when the target installer went to look for items using properly-cased paths they didn't match, so it didn't add them to the targets.

efirestone commented:
@segiddins FYI. Sorry about introducing this in the first place.

segiddins commented on CHANGELOG.md:
```diff
@@ -37,6 +37,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Properly add resource files to resources build phase.
```
needs to go in a `Master` section

efirestone replied:
Good catch, fixed.

segiddins commented:
Could you add a test for different cases?

efirestone commented:
I added checks to (case-sensitively) look at the path as well as the fact that the file ref exists, which checks that this won't regress. I also added a check that the build resources target includes `en.lproj`. Both of these would have caught this issue.

segiddins commented:
Great, thanks!

nkibler commented:
When can we expect this to be released? This is blocking us from moving off 0.39.0, which is causing us issues that #4635 solves.

cnoon commented:
Ping here as well. This issue is blocking us from being able to switch over to test with the beta releases.

segiddins commented:
https://guides.cocoapods.org/using/unreleased-features.html

cnoon commented:
Thanks for your quick reply @segiddins...much appreciated. Unfortunately for my current situation, it's not realistic for me to ask 20+ engineers to switch over to the unreleased version. I merely wanted to call attention to the fact that this particular bug is a blocker for us. I figured the more info coming your way about the betas, the better.

We have no expectations here, I realize how hectic scheduling big releases can be. We greatly appreciate all the hard work your fantastic team does every day! 🍻
